pkg/cmd/scan.go

```diff
@@ -225,6 +227,9 @@ func scanRun(opts *pkg.ScanOptions) error {
 	}()
 
 	analysis, err := ctl.Run()
+	if !opts.DisableTelemetry {
+		telemetry.SendTelemetry(store)
+	}
```

Comment:
I should revert this since we don't send any error related field in telemetry data